### PR TITLE
[excel, outlook, powerpoint, word] (Script Lab) Update Script Lab documentation to omit gist sharing functions

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR covers some potential upcoming changes to Script Lab.

### CODE REVIEW

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT: This PR is now ready for review.

**Comment by AlexJerabek** [comment]
Tagging @wandyezj for awareness.

File: docs/overview/explore-with-script-lab.md [hunk 1]
```diff
@@ -22,36 +22,31 @@ See Script Lab in action in this one-minute YouTube video.
...
-
-![Sharing options.](../images/script-lab-share.jpg)
+To call preview APIs within a snippet, you need to update the snippet's libraries to use the beta content delivery network (CDN) (`https://appsforoffice.microsoft.com/lib/beta/hosted/office.js`) and the preview type definitions `@types/office-js-preview`. Additionally, some preview APIs are only accessible if you've signed up for the [Microsoft 365 Insider program](https://insider.microsoft365.com) and are running an Insider build of Office.
 
-### Import snippets
+### Save and share snippets
 
-You can import a snippet into Script Lab either by specifying the URL to the public [GitHub gist](https://gist.github.com) where the snippet YAML is stored or by pasting in the complete YAML for the snippet. This feature may be useful in scenarios where someone else has shared their snippet with you by either publishing it to a GitHub gist or providing their snippet's YAML.
+By default, snippets that you open in Script Lab are saved to your browser cache or local storage. To save a snippet permanently, select **Copy** and paste the resulting clipboard content into a new .yml file.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:47]: Can't we still mention GitHub gist as a storage/sharing option? It just won't be an import option.

```suggestion
By default, snippets that you open in Script Lab are saved to your browser cache or local storage. To save a snippet permanently, select **Copy** and paste the resulting clipboard content into a new .yml file. Use this to share snippets with colleagues or provide code for community sites, such as Stack Overflow.
```

File: docs/overview/explore-with-script-lab.md [hunk 2]
```diff
@@ -22,36 +22,31 @@ See Script Lab in action in this one-minute YouTube video.
...
 
-![Import snippet option.](../images/script-lab-import-snippet.jpg)
+To import a snippet into Script Lab, select **Import** from the menu and paste in the complete YAML for the snippet. Use this to share snippets with colleagues or provide code for community sites, such as Stack Overflow.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:49]:
```suggestion
To import a snippet into Script Lab, select **Import** from the menu and paste in the complete YAML for the snippet.
```

File: docs/overview/explore-with-script-lab.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/overview/explore-with-script-lab.md:47]: I want to avoid mentioning GitHub gists because it's a specific (Microsoft-owned) storage solution.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**